core-services/image-mirroring/knative/mapping_knative_nightly_quay

```diff
@@ -11,11 +11,11 @@ registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-reques
 registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:latest
 registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:latest
 registry.ci.openshift.org/openshift/knative-nightly:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:latest
-registry.ci.openshift.org/openshift/openshift-serverless-nightly:openshift-knative-operator quay.io/openshift-knative/openshift-knative-operator:latest
-registry.ci.openshift.org/openshift/openshift-serverless-nightly:knative-operator quay.io/openshift-knative/knative-operator:latest
-registry.ci.openshift.org/openshift/openshift-serverless-nightly:knative-openshift-ingress quay.io/openshift-knative/knative-openshift-ingress:latest
-registry.ci.openshift.org/openshift/openshift-serverless-nightly:metadata-webhook quay.io/openshift-knative/metadata-webhook:latest
-registry.ci.openshift.org/openshift/openshift-serverless-nightly:serverless-stop-bundle quay.io/openshift-knative/serverless-stop-bundle:latest
+registry.ci.openshift.org/knative/openshift-serverless-nightly:openshift-knative-operator quay.io/openshift-knative/openshift-knative-operator:latest
+registry.ci.openshift.org/knative/openshift-serverless-nightly:knative-operator quay.io/openshift-knative/knative-operator:latest
+registry.ci.openshift.org/knative/openshift-serverless-nightly:knative-openshift-ingress quay.io/openshift-knative/knative-openshift-ingress:latest
+registry.ci.openshift.org/knative/openshift-serverless-nightly:metadata-webhook quay.io/openshift-knative/metadata-webhook:latest
+registry.ci.openshift.org/knative/openshift-serverless-nightly:serverless-stop-bundle quay.io/openshift-knative/serverless-stop-bundle:latest
```

Comment:
A few comments:
a) Images that use the knative namespace are listed [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.11.yaml#L37-L99). Do we need to map tags: serverless-index, serverless-stop-bundle too?
b) Why do we need to map the above images? Where do we use the S-O quay.io ones?
c) Generally it is confusing to use `knative` with registry.ci.openshift.org and  `openshift-knative` with quay.io. 
Not to mention that test images are still under openshift namespace eg. `registry.ci.openshift.org/openshift/knative-nightly:knative-serving-test-httpproxy`

Reply:
For b), you can find them where are used in S-O repo.
For a) c), I think it should have been discussed in https://github.com/openshift/release/pull/28552